File: TSP/sample/python/TravelingSalesman.py
import sys

N = int(raw_input())
posX = [];
posY = [];

for i in range(N):
	p = map(int,raw_input().split())
	posX.append(p[0])
	posY.append(p[1])

for i in range(N):
    print i

sys.stdout.flush()

File: TravelingSalesman/sample/cpp/TravelingSalesman.cpp
#include <iostream>
#include <vector>
using namespace std;

int main () {

    int N; 
    cin >> N;

    vector < int > posX(N), posY(N);
    for (int i = 0; i < N; i++) {
        cin >> posX[i] >> posY[i];
    }

    for (int i = 0; i < N; i++) {
        cout << i << endl;
    }

    return 0;
    
}
